frontera/contrib/backends/remote/codecs/json.py

```diff
@@ -45,6 +75,10 @@ def __init__(self, request_model, *a, **kw):
         self.send_body = kw.pop('send_body', False)
         super(Encoder, self).__init__(request_model, *a, **kw)
 
+    def encode(self, obj):
+        converted = _convert(obj)
```

Comment:
Won't this error occur for headers, cookies or in fact all dict type objects? Right now issues are only reported for meta.

Also any suggestions to make this efficient for msgpack?